### ISSUE TITLE
Create simple SVM classifier in Tensorflow

### ISSUE THREAD

**Issue by jdpigeon** [NeuroTechX/neurodoro#2]
So, we know that we're going to have to collect a bunch of data and go deep to make something really impressive, but for our debut in the app store we might as well include a 'decent enough' classifier so people can get some benefit.

The cross-validation accuracy of a SVM applied to denoised PSD data isn't terrible. Why don't we just do that?

- Select best features (research says alpha, beta, and alpha/beta ratio). Also try some kind of mathematical feature selection?
- Collect some higher quality training data with new version of app and cognitive testing screener
- Tune hyperparameters and get decent accuracy in sci-kit learn
- Implement the same SVM in Tensorflow
- Port into app
